docs/onboarding/02-digital-twin-definition-language.md

```diff
@@ -3,6 +3,7 @@
 DTDL is the schema by which models are described. Its important to understand how models are described and what constructs are supported. DTDL is to Azure Digital Twins what DDL (Data Definition Language) is to SQL.
 
 - [Concepts](https://docs.microsoft.com/en-us/azure/digital-twins/concepts-models) - ~11 min. Covers basic concepts of DTDL language
+- [Marker tags](https://docs.microsoft.com/en-us/azure/digital-twins/how-to-use-tags) ~10 min. Covers how to define tags, how to provision tag markers to your twins and how to query filtering by tags.

```

Comment:
Should we move this under model management?

Reply:
Actually I think this topic should be part of "Concepts". It is just another way to define a property -at design time-. That's why I believe it should be right there!